keyboards/4by3/rules.mk

```diff
@@ -0,0 +1,9 @@
+MCU 				= atmega32u4
+BOOTLOADER 			= caterina
+
+EXTRAKEY_ENABLE 	= yes
+NKRO_ENABLE 		= yes
+CONSOLE_ENABLE		= yes
+COMMAND_ENABLE		= yes
+
+LAYOUTS				= horizontal vertical_right vertical_left
```

Comment:
The `LAYOUTS` rule is only for the [Community Layouts feature](https://docs.qmk.fm/#/feature_layouts). As none of the listed options are currently valid choices, this line should be deleted.

```suggestion
```